ossapi/ossapiv2.py

```diff
@@ -18,10 +18,11 @@
 from ossapi.models import (Beatmap, BeatmapUserScore, ForumTopicAndPosts,
     Search, CommentBundle, Cursor, Score, BeatmapSearchResult,
     ModdingHistoryEventsBundle, User, Rankings, BeatmapScores, KudosuHistory,
-    Beatmapset, MostPlayedBeatmap, Spotlight, Spotlights, _Event, Event)
+    Beatmapset, MostPlayedBeatmap, Spotlight, Spotlights, _Event, Event, 
```

Comment:
please enable trimming trailing whitespace on save (should be a setting in all popular editors), this line has a trailing space after it